src/main/java/nl/tudelft/ti2206/group9/level/InternalTicker.java

```diff
@@ -46,25 +51,30 @@ private InternalTicker(final Timer t) {
 	 * Thread method.
 	 */
 	public void run() {
-		synchronized (GameWindow.LOCK) {
-			final Timer newTimer = new Timer();
-	
-			try {
-				step(); // First, perform tick.
+		Platform.runLater(new Runnable() {
+			@Override
+			public void run() {
```

Comment:
Actually... I can't run the game at the moment?

    Exception in thread "JavaFX Application Thread" java.lang.Error: Unresolved compilation problem: 
	The method run() of type new Runnable(){} must override a superclass method

	at nl.tudelft.ti2206.group9.level.InternalTicker$1.run(InternalTicker.java:56)
	at com.sun.javafx.application.PlatformImpl.lambda$null$170(PlatformImpl.java:295)
	at com.sun.javafx.application.PlatformImpl$$Lambda$48/610250873.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$171(PlatformImpl.java:294)
	at com.sun.javafx.application.PlatformImpl$$Lambda$46/237061348.run(Unknown Source)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$145(WinApplication.java:101)
	at com.sun.glass.ui.win.WinApplication$$Lambda$36/2117255219.run(Unknown Source)
	at java.lang.Thread.run(Thread.java:745)